vxquery-core/src/main/java/org/apache/vxquery/metadata/VXQueryCollectionDataSource.java

```diff
@@ -85,6 +87,16 @@ public void setPartitions(String[] collectionPartitions) {
     public int getPartitionCount() {
         return collectionPartitions.length;
     }
+    
+    public String getTag()
+    {
+        return this.tag;
+    }
+    
+    public void setTag(String tag)
```

Comment:
Should we just create a separate constructor where the tag is added? Then tag can be final.